include/spirv-tools/optimizer.hpp

```diff
@@ -77,6 +77,11 @@ class Optimizer {
   // method.
   Optimizer& RegisterPass(PassToken&& pass);
 
+  // Registers passes that attempt to legalize the generated code.
+  // This sequence of passes is subject to constant review and will change
```

Comment:
It's worth saying these are passes used to legalize code originally generated by legacy HLSL.  Also say where to find a more current list, i.e. in Glslang

Reply:
Done.